content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
 ---
 
 <!-- overview -->
-This page describes common concepts in the Kubernetes API.
-
+This page describes Kubernetes' API and conventions for interacting with it.
 
 <!-- body -->
 The Kubernetes API is a resource-based (RESTful) programmatic interface provided via HTTP. It supports retrieving, creating,
-updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET), includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. It also supports efficient change notifications on resources via "watches" and consistent lists to allow other components to effectively cache and synchronize the state of resources.
+updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET). The API includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. Kubernetes also supports efficient change notifications on resources via "watches" and consistent lists, so that API clients can effectively cache, track, and synchronize the state of resources.
```

Comment:
For such a long line, do you have any suggestions how can I easily find out what have been changed?
Are these changes absolutely necessary?

Reply:
The original has a long line, and GitHub's diff detection isn't great for documentation.
If you fetch this PR locally you can run eg
`git diff --word-diff=color --word-diff-regex=.` which shows character-by-character changes.

Reply:
Is replacing "other components" with "API clients" the correct substitution?
I think "other components" refers to components in the cluster that act as "API clients". When I read "API clients" I think of external HTTP clients.

`to allow other cluster components to effectively cache and synchronize the state of resources`.

Reply:
Maybe “API clients (including other control plane components)”? A watcher can be part of the control plane, or it can be a Pod, or it can be `kubectl`.

Reply:
I've omitted this change from the latest revision.